bql/semantic/semantic.go

```diff
@@ -122,6 +128,33 @@ type GraphClause struct {
 	OTemporal        bool
 }
 
+// ConstructClause represents a singular clause within a construct statement.
+type ConstructClause struct {
```

Comment:
Despite the name this is just the info on the construct right? Just want to make sure I got it right :)